OpenRA.Editor/Form1.cs

```diff
@@ -318,6 +318,107 @@ void PopulateActorOwnerChooser()
 			surface1.NewActorOwner = ((PlayerReference)actorOwnerChooser.SelectedItem).Name;
 		}
 
+		void SetupMap(Map map, int width, int height, int left, int right, int top, int bottom, bool resize = false)
+		{
+			var badValue = false;
+
+			if (width < 10 || width > 550 || height < 10 || height > 550)
```

Comment:
I am not sure about those bounds. In theory a 8x8 map or a 640x640 map would be totally fine as long as it got cordons and the boundaries are dividable by 8. See https://github.com/OpenRA/OpenRA/wiki/Mapping#file-format Maybe seek some inspiration from the checks at https://github.com/OpenRA/OpenRA/blob/master/OpenRA.Mods.RA/Lint/CheckMapCordon.cs instead.